packages/utilities/tests/omitKeysFromObject.test.ts

```diff
@@ -0,0 +1,43 @@
+import { omitKeysFromObject } from '../src/lib/omitKeysFromObject';
+
+describe('omitKeysFromObject', () => {
```

Comment:
There should be a test checking that `source` and `result` are different objects, and that `source` isn't mutated.